bff-service/.env

```diff
@@ -0,0 +1,2 @@
+product=https://y8cv6h0sql.execute-api.eu-west-1.amazonaws.com/dev/products
```

Comment:
Hi! Just an advice: .env files should be added to .gitignore, if not - you can expose your secret data  :)

Reply:
Hi! Thanks!